modules/internal/ChapelDistribution.chpl

```diff
@@ -404,25 +404,44 @@ module ChapelDistribution {
 
       if !dataSorted && preserveInds {
         var _inds = inds;
-        return bulkAdd_help(_inds, dataSorted, isUnique); 
+        return bulkAdd_help(_inds, dataSorted, isUnique);
       }
       else {
         return bulkAdd_help(inds, dataSorted, isUnique);
       }
     }
 
-    proc bulkAdd_help(inds: [?indsDom] index(rank, idxType), 
+    proc bulkAdd_help(inds: [?indsDom] index(rank, idxType),
         dataSorted=false, isUnique=false){
       halt("Helper function called on the BaseSparseDomImpl");
 
       return -1;
     }
 
-    inline proc _grow(size: int){
+    // TODO: Would ChapelArray.resizeAllocRange() be too expensive?
+    //       - would have to put the conditional outside of the call
+    /* Grow domain if necesary */
+    inline proc _grow(size: int, factor=arrayAsVecGrowthFactor) {
```

Comment:
What's the motivation for adding this factor argument, changing it from `2`?  (esp since it's not overridden anywhere).  Does it result in better performance?